### PR TITLE
Updates and bug fixes

### PR DESCRIPTION
**Pull request by Kdreval**
This PR has fixes for the following issues:
- #70 
- #71 
- #72
- #73
- #68 
- #67

### CODE REVIEW

**Review by kmcoyle — APPROVED** [review]
Seems good to me!

**Comment by Kdreval** [comment]
Thanks a bunch @kmcoyle !

File: README.md [hunk 1]
```diff
@@ -1,46 +1,13 @@
...
-* Other resources
-
-## Installation
-
-GAMBLR.data is an open-source package. It can be easily installed directly from GitHub:
-
-```
-devtools::install_github("morinlab/GAMBLR.data", repos = BiocManager::repositories())
-```
+![](https://github.com/morinlab/GAMBLR.data/actions/workflows/build_check.yml/badge.svg)
```
Comment (vladimirsouza) [inline README.md:1]: Thank you very much for this PR.

The only thing that I would like to mention is that this badge is not shown to me.

Comment (Kdreval) [reply README.md:1]: This PR was reviewed and merged. Please create an issue if there is some bug or something needs attention

**Pull request merged**